test/os/ui/draw/draw.test.js

```diff
@@ -0,0 +1,41 @@
+goog.require('ol.Feature');
+goog.require('ol.geom.Polygon');
+goog.require('ol.proj');
+goog.require('os.map');
+goog.require('os.ui.draw');
+
+describe('os.ui.draw', function() {
+  window.localStorage.clear();
+  os.net.RequestHandlerFactory.addHandler(os.net.SameDomainHandler);
```

Comment:
This should already be done, and you're also not using requests below so why is it here?